components/avatar/avatar.css

```diff
@@ -0,0 +1,98 @@
+@import "/common.css";
+
+/* .avatar-container {
+  box-sizing: border-box;
+  flex-grow: 1;
+  margin-left: 5rem;
+} */
+
+.avatar-header {
+  font-size: 2rem;
+}
+
+.avatar-content {
+  font-size: 1.5rem;
+  margin: 0.5rem;
+  margin-bottom: 2rem;
+}
+
+.avatar-list {
+  display: flex;
+  align-items: center;
+  margin-left: 2rem;
+  gap: 1.4rem;
+  margin-bottom: 2rem;
+}
+
+.avatar-list-item {
+  height: 2rem;
+  display: inline-block;
+}
+
+.avt-img {
+  border-radius: 50%;
+  height: 3rem;
+}
+
+.avatar {
+  border-radius: 50%;
+  margin-left: 3rem;
+}
+
+.avatar-xs {
+  max-height: 1rem;
+  max-width: 1rem;
+}
+
+.avatar-sm {
+  max-height: 2rem;
+  max-width: 2rem;
+}
+.avatar-xd {
+  max-width: 3rem;
+  max-height: 3rem;
+}
+
+.avatar-md {
+  max-height: 4rem;
+  max-width: 4rem;
+}
+
+.avatar-lg {
+  max-height: 5rem;
+  max-width: 5rem;
+}
+
+.avatar-badge {
+  margin-top: 1.5rem;
+}
+.badge-img {
+  width: 100%;
+  max-height: fit-content;
+}
+
+.badge {
+  display: inline-flex;
+  position: relative;
+  width: 3rem;
+  margin: 1.6rem;
+}
+
+.badge-status {
+  position: absolute;
+  height: 1rem;
+  width: 1rem;
+  border-radius: 50%;
+  left: 5rem;
+  top: 2rem;
+}
+.badge-online {
```

Comment:
stay consistent in spacing. if you are giving 1 line break between each class then give it the same way everywhere

Reply:
Thanks for suggestion.